app/assets/stylesheets/multi_page.scss

```diff
@@ -0,0 +1,27 @@
+$base        : #AAAAAA;
+$secondary-color: white;
+
+.flash {
+  background-color: maroon;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.